solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRRescorer.java

```diff
@@ -186,18 +191,40 @@ public void scoreFeatures(IndexSearcher indexSearcher,
         docBase = readerContext.docBase;
         scorer = modelWeight.scorer(readerContext);
       }
-      scoreSingleHit(indexSearcher, topN, modelWeight, docBase, hitUpto, hit, docID, scoringQuery, scorer, reranked);
+      scoreSingleHit(
+          indexSearcher,
+          topN,
+          modelWeight,
+          docBase,
+          hitUpto,
+          hit,
+          docID,
+          scoringQuery,
+          scorer,
+          reranked);
       hitUpto++;
     }
   }
 
-  protected static void scoreSingleHit(IndexSearcher indexSearcher, int topN, LTRScoringQuery.ModelWeight modelWeight, int docBase, int hitUpto, ScoreDoc hit, int docID, LTRScoringQuery rerankingQuery, LTRScoringQuery.ModelWeight.ModelScorer scorer, ScoreDoc[] reranked) throws IOException {
+  protected static void scoreSingleHit(
+      IndexSearcher indexSearcher,
+      int topN,
+      LTRScoringQuery.ModelWeight modelWeight,
+      int docBase,
+      int hitUpto,
+      ScoreDoc hit,
+      int docID,
+      LTRScoringQuery rerankingQuery,
+      LTRScoringQuery.ModelWeight.ModelScorer scorer,
+      ScoreDoc[] reranked)
+      throws IOException {
     final FeatureLogger featureLogger = rerankingQuery.getFeatureLogger();
     // Scorer for a LTRScoringQuery.ModelWeight should never be null since we always have to
```

Comment:
Can we make a block comment here, I think that would collapse some of the line fragments better.